deps-version

```diff
@@ -1 +1 @@
-1.1
+1.1.1
```

Comment:
concourse auto increments the minor version (third number) so to bump this manually this should be `1.2`